android-core/src/main/java/com/mparticle/MParticleOptions.java

```diff
@@ -55,6 +55,7 @@ public class MParticleOptions {
     private DataplanOptions mDataplanOptions;
     private Map<Class, List<Configuration>> mConfigurations = new HashMap();
     private List<SideloadedKit> sideloadedKits = new ArrayList<>();
+    private List<Integer> filteredKits = new ArrayList<>();
```

Comment:
I'm sorry I didn't look at the changes they suggested for android. They named this disabledKits in iOS and filteredKits for Android? We should align on one and I'm biased but I think it should be "disabledKits"

Reply:
I agree with Brandon here.  Let's use the term `disable` throughout.  It is much more clear in its purpose than filter, which is a bit vague.